lib/Backend/Backend.cpp

```diff
@@ -40,7 +40,7 @@ TraceInfo Backend::buildManualTraceInfo(Function *F) const {
 
 void Backend::autoInstrument(TraceInfo &traceInfo, IRFunction *IR) const {
   if (getTraceEventDataSize() == 0) {
-    GLOW_UNREACHABLE("Auto instrumentation not supported on this backend");
+    LOG(ERROR) << "Auto instrumentation not supported on this backend";
     return;
```

Comment:
I see this return was here before, but is it still necessary?

Reply:
Will remove, also this should have been LOG(FATAL) oops

Reply:
Actually wait no, this is right. If autoInstrument isn't supported, we should just log that and keep going

Reply:
Ok, but that's different than previous behavior right? `GLOW_UNREACHABLE()` would always abort.

Reply:
Yes this is different behavior we probably shouldn't kill the process due to this.